src/signals/incident/definitions/wizard-step-2-vulaan/afval.constants.ts

```diff
@@ -0,0 +1 @@
+export const CONTAINER_NOT_ON_MAP_TYPE_NAME = 'not-on-map';
```

Comment:
I would place this in `...ContainerSelect/constants.ts`  the afval configuration is to be moved to the backend.